README.md

```diff
@@ -35,8 +36,19 @@ npm install fiori-fundamentals --save
 
 > NOTE: We only distribute the [fonts, icons and CSS](https://github.com/SAP/fundamental/tree/master/dist) and [the SASS source](https://github.com/SAP/fundamental/tree/master/scss), not the full project or HTML for specific components.
 
+
+### LESS compatible Fiori Fundamentals
+
```

Comment:
This is way too much real estate to provide to LESS.
remove this text, add it to a wiki page, and link to it here.

Reply:
additionally, below this, perhaps have another heading here for icons, and a link to the icon wiki